diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/BlockStructureBraces.kt

```diff
@@ -155,26 +153,43 @@ class BlockStructureBraces : Rule("block-structure") {
             CLASS, OBJECT_DECLARATION -> node.findChildByType(CLASS_BODY)!!.findChildAfter(LBRACE, WHITE_SPACE)
             else -> node.findChildByType(BLOCK)?.findChildAfter(LBRACE, WHITE_SPACE)
         }
-        if (nodeText != null && !nodeText.text.contains("\n".toRegex())) {
-            BRACES_BLOCK_STRUCTURE_ERROR.warnAndFix(configRules, emitWarn, isFixMode, "incorrect line after opening brace",
-                    nodeText.startOffset) {
+        if (newNode != null && !newNode.text.contains("\n")) {
+            BRACES_BLOCK_STRUCTURE_ERROR.warnAndFix(configRules, emitWarn, isFixMode, "incorrect same line after opening brace",
+                    newNode.startOffset) {
+            }
+        }
+    }
+
+    private fun checkMidBrace(node: ASTNode){
+        val allMiddleSpace = when(node.elementType){
+            TRY -> {
+                node.findAllNodesWithSpecificType(FINALLY).map { it.prevSibling { type -> type.elementType == WHITE_SPACE } } +
+                node.findAllNodesWithSpecificType(CATCH).map { it.prevSibling { type -> type.elementType == WHITE_SPACE } }
+            }
+            else -> node.findAllNodesWithSpecificType(THEN).map { it.nextSibling { type -> type.elementType == WHITE_SPACE } }
+        }
+        allMiddleSpace.forEach {
+            if (it!!.text.contains("\n")){
```

Comment:
Assumption that this node is not null may not be right: 
1) there could be no white space between brace and keyword (`try { }finally { }`)
2) Therefore, when you find `prevSibling` of type `WHITE_SPACE` you might get not the node you are expecting.

I suggest to just map to `treePrev` and `takeIf` if it's type is `WHITE_SPACE`. Then you should check nullability when checking for newline.